src/search_algo.hpp

```diff
@@ -1158,14 +1200,15 @@ iterateMatchesFullSerial(TLocalHolder & lH)
     for (auto it = lH.matches.begin(), itEnd = lH.matches.end(); it != itEnd; ++it)
     {
         // create blastmatch in list without copy or move
-        record.matches.emplace_back(lH.qryIds [it->qryId / seqan::qNumFrames(TGlobalHolder::blastProgram)],
-                                    lH.gH.indexFile.ids[it->subjId / seqan::sNumFrames(TGlobalHolder::blastProgram)]);
+        bool fwd =  it->fwd;
```

Comment:
As I said, I think this can be computer here. Also maybe move it down to where it is used (l 1222)?.